images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -115,7 +114,9 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-GoogleCloudSDKVersion)
 )
 
-$markdown += New-MDHeader "Rust packages:" -Level 3
+$markdown += New-MDHeader "Rust" -Level 3
+$markdown += New-MDParagraph -Lines "Version: $(Get-RustVersion)"
```

Comment:
Let's replace `Version:` on `Rust` like we have done for MacOS

Reply:
Done.